python/tritonserver/_api/_model.py

```diff
@@ -275,31 +251,12 @@ def infer(
         if inference_request is None:
             inference_request = InferenceRequest(model=self, **kwargs)
 
-        if (inference_request.response_queue is not None) and (
-            not isinstance(inference_request.response_queue, queue.SimpleQueue)
-        ):
-            raise InvalidArgumentError(
-                "queue.SimpleQueue must be used for response iterator"
-            )
-
+        # The 'request' is referencing pointers managed by the 'inference_request', so
```

Comment:
see above - I think this more logically belongs with the comments for the Response Iterator - or could be omitted

Reply:
sure, updated.